tiny_tokenizer/tiny_tokenizer_token.py

```diff
@@ -65,12 +79,16 @@ def feature(self):
         feature = []
         if self.postag is not None:
             feature.append(self.postag)
+        if self.postag1 is not None:
+            feature.append(self.postag1)
         if self.postag2 is not None:
             feature.append(self.postag2)
-        if self.conj_type is not None:
-            feature.append(self.conj_type)
-        if self.conj_form is not None:
-            feature.append(self.conj_form)
+        if self.postag3 is not None:
+            feature.append(self.postag3)
+        if self.inflection is not None:
+            feature.append(self.inflection)
+        if self.conjugation is not None:
+            feature.append(self.conjugation)
         if self.origin_form is not None:
             feature.append(self.origin_form)
         if self.yomi is not None:
```

Comment:
There isn't about `normalized_form`. Is it OK?

Reply:
Nice catch!